docs/accept-a-maximum-number-of-orders-per-day/script.liquid

```diff
@@ -279,7 +307,9 @@
     {% endfor %}
 
     {% if inventory_levels_to_restore != blank %}
-      {% action "cache", "del", cache_key %}
+      {% for cache_key in cache_keys %}
+        {% action "cache", "del", cache_key %}
+      {% endfor %}
```

Comment:
I _think_ there's a hole here, in the event that multiple cache pages are saved, but _afterwards_ the total number of inventory items drops below 2000. In that event, there wouldn't be evidence that multiple cache pages exist, which means that subsequent pages wouldn't be deleted.

Those cache pages will expire, and the time-based cache key means they wouldn't ever be used, sooo I'm fine with any of the following:

1. Solve it with documentation, noting that there's a hole and it's not an issue.
2. Use setex with the cache action, and have each cached document last .. what, 48hrs? Feels risky, though.
3. Look up the cache pages before deleting them, and follow the linked list to delete all linked cache pages.

Let me know if this makes sense, and also let me know if I'm completely off-base about this being an issue. :D

Reply:
The cache delete is only during the restore loop, so it _should_ always get all of them.

I guess there could be a straggler if someone disabled the restore options _after_ the cache was set. But then, on the next cache set, the last group cache (assuming less inventory items) wouldn't have a "next_key".

Reply:
Ahhh, okay, that is a more specific situation than I was thinking. Thanks for spelling it out. :) Am good with this then.